app/controllers/dashboard_controller.rb

```diff
@@ -1,5 +1,7 @@
 class DashboardController < ApplicationController
-  before_action :authenticate
   def show
+    geolocation = GoogleGeolocationService.new
```

Comment:
Uses the current user's lat and long to find their location when finding the nearest restaurant.